nova/tests/fixtures/nova.py

```diff
@@ -201,6 +201,11 @@ def delete_stored_logs(self):
         # https://github.com/testing-cabal/fixtures/pull/42 merges
         self.logger._output.truncate(0)
 
+class DisableSAPQuotaEngineCacheFixture(fixtures.Fixture):
+    def setUp(self):
+        super(DisableSAPQuotaEngineCacheFixture, self).setUp()
+        self.useFixture(fixtures.MonkeyPatch('nova.quota.SAPQuotaEngine._cache', {}))
```

Comment:
Is there no noop-cache driver we could use?

Reply:
There is, but we can't use it, because cache is disabled by default and then the backend doesn't matter. We also don't need the full backend, because Nova wraps it into CacheClient.